apps/morph_check

```diff
@@ -132,24 +137,29 @@ def log_msg(ok, msg, color=False):
     L.log(LOG_LEVELS[ok],
           '%35s %s' + CEND, msg, CGREEN + 'PASS' if ok else CRED + 'FAIL')
 
+    if summary_dict is not None:
+        summary_dict[msg] = 'PASS' if ok else 'FAIL'
+
 
 def test_file(f, config):
     '''Run tests on a morphology file'''
 
     L.info('Check file %s...', f)
 
+    summary_ = {}
```

Comment:
Ah, I would make __main__ shorter and just call main().  Makes it easier to test, too